scripts/build-all-bottles.sh

```diff
@@ -0,0 +1,47 @@
+#! /usr/bin/env nix-shell
+#! nix-shell shell.nix -i bash
+
+# SPDX-FileCopyrightText: 2022 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+set -euo pipefail
+
+if [ -z "$1" ] ; then
+    echo "Please call this script with the name of the OS for which to build the bottles."
+    exit 1
+fi
+
+# we don't bottle meta-formulas that contain only services
+formulae=("tezos-accuser-011-PtHangz2" "tezos-accuser-012-Psithaca" "tezos-admin-client" "tezos-baker-011-PtHangz2" "tezos-baker-012-Psithaca" "tezos-client" "tezos-codec" "tezos-endorser-011-PtHangz2" "tezos-node" "tezos-sandbox" "tezos-signer")
+
+# we filter out only the formulas that were changed and so need to be rebuilt
+changed_formulae=()
+for f in "${formulae[@]}"; do
+  if ./scripts/check-bottle-built.sh "$f" "$1"; then
```

Comment:
Does it make sense to move this check to the `for` loop below (and traverse over `formulae` in it) to reduce the amount of code in this script a bit?